docs/release-notes/index.md

```diff
@@ -12,3 +12,26 @@ permalink: /release-notes
 ### v1.0.0 (released 2019-04-09)
 
 * First GA, production-ready release.
+
+## v2.0
+
+### v2.0.0 (released tbd)
+
+* Request specs use backwards-incompatible sets-arg-as syntax. Script to upgrade [here](https://github.com/square/spincycle/util/reformat-yaml/).
```

Comment:
You should also describe the point of the change, i.e. that you can now rename arguments set by a node.